docs/source/specification.md

```diff
@@ -22,102 +27,147 @@ For example:
         └── ...
 ```
 
-The project folder may have any name descriptive of the project (that does not include spaces). Within the 
-project folder, data is separated into `rawdata` and `derivatives`. `rawdata` contains the raw
-data acquired during data collection. `derivatives` includes any processed data (e.g. spike sorting or pose estimation). 
-The subject / session / datatype folder structure within derivatives should
-match the rawdata folder whenever possible.
+### Basic principles
+
+* The project folder ***may*** have any name descriptive of the project but this ***must not*** contain spaces.
```

Comment:
For me personally, I prefer keywords of required, optional and recommended in italics only. I find the words must / must not with bold a little commanding / annoying. This mostly comes from my first memory of reading BIDS, thinking that it is already a pain that I am going to re-organise my whole directory structure and they are going to say I MUST? do xyz? But, maybe most people don't mind and I have some unresolved issues with being told what to do 😅

Reply:
It's now italics